src/cocotb/share/makefiles/Makefile.inc

```
@@ -93,6 +93,12 @@ endef
 SIM_BUILD ?= sim_build
 export SIM_BUILD
 
+ifeq ($(NO_SIM_LOG),)
+    SIM_LOG_TEE := 2>&1 | tee $(SIM_BUILD)/sim.log
+else
+    SIM_LOG_TEE :=
+endif
+
```

Rather than using a flag, why not introduce a variable the user can overload to whatever they please?
```suggestion
SIM_OUTPUT_REDIRECT ?= 2>&1 | tee $(SIM_BUILD)/sim.log

```

---

Yeah, good call.